core/api.txt

```diff
@@ -428,6 +428,7 @@ ion-datetime,css-prop,--background-rgb
 ion-datetime,css-prop,--title-color
 ion-datetime,css-prop,--wheel-fade-background-rgb
 ion-datetime,css-prop,--wheel-highlight-background
+ion-datetime,css-prop,--wheel-highlight-border-radius
```

Comment:
The border radius is not showing up on iOS anymore: 
![image](https://github.com/ionic-team/ionic-framework/assets/2721089/20b47385-6e65-43cf-868e-dd9a7953dbda)

Reply:
Fixed https://github.com/ionic-team/ionic-framework/pull/28654/commits/294c3043a3ce7bbd7e1e91920d5bdd35a5ff0352